### PR TITLE
use channel name or tags to filter `!libs`

### PR DESCRIPTION
**Pull request by cuppachino**
This should mean when you're in tft-dev and run !libs you get a list of tft libraries. Any extra arguments are appended to the topic list instead of replying with an error.

### CODE REVIEW

File: src/RiotAPILibraries.ts [hunk 1]
```diff
@@ -50,6 +50,13 @@ interface LibraryDescription {
 }
 
 export default class RiotAPILibraries {
+    static CHANNEL_TOPICS = {
+        lol: ["lol", "v4"],
+        lcu: ["lcu"],
+        tft: ["tft"],
+        lor: ["lor"]
+    } as const;
+
```
Comment (Token07) [inline src/RiotAPILibraries.ts:59]: Maybe consider moving this to shared_settings.json

**Review by Token07 — APPROVED** [review]
Seems to work on my test env

File: src/RiotAPILibraries.ts (hunk 1 above)
Comment (cuppachino) [reply src/RiotAPILibraries.ts]: good catch 👍

**Comment by cuppachino** [comment]
Is there anything else I can do before merging?

**Pull request merged**